web/src/app/app.component.ts

```diff
@@ -1,16 +1,38 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, NgZone } from '@angular/core';
+import { Router } from '@angular/router';
 import { environment } from '../environments/environment';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { AppStateService } from './services/app-state.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   public title = 'TribeDashboard';
   public version: string;
 
-  constructor() {
+  constructor(
+    public router: Router,
+    public afAuth: AngularFireAuth,
+    public zone: NgZone,
+    public appState: AppStateService,
+  ) {
     this.version = environment.version;
+    this.router = router;
```

Comment:
this is not needed, it will automatically be mapped to a class variable

Reply:
Added for testing the access specifiers. Will remove it.